libs/tokens/src/utils/tokenMapToListWithLogo.ts

```diff
@@ -0,0 +1,12 @@
+import { TokenWithLogo } from '@cowprotocol/common-const'
+
+import { TokensMap } from '../types'
+
+/**
+ * Convert a tokens map to a list of tokens and sort them alphabetically
+ */
+export function tokenMapToListWithLogo(tokenMap: TokensMap): TokenWithLogo[] {
+  return Object.values(tokenMap)
+    .sort((a, b) => (a.symbol > b.symbol ? 1 : -1))
```

Comment:
Does it mater it they are lower cased for the comparison?

Reply:
I don't see any difference, maybe because the majority of tokens have uppercased symbols